Makefile

```diff
@@ -250,6 +250,24 @@ telemetry-mocks: ## Creates mock files.
 store-layers: ## Generate layers for the store
 	$(GO) generate $(GOFLAGS) ./store
 
+migration-prereqs: ## Builds prerequisite packages for migrations
+ifeq ($(PLATFORM),Windows)
+	@powershell ./exe/migration_prereqs.ps1
+else
+	@./scripts/migration-prereqs
+endif
```

Comment:
Let's use the standard `go get` method to install tools and pin the dependencies in go.tools.mod. It avoids different tool versions and any custom shell/powershell scripts to install these tools.

The only exception was `golangci-lint` where the README explicitly discouraged from using go-get to install the tool.

Reply:
oh good point, on here https://github.com/golang-migrate/migrate/tree/master/cmd/migrate i didn;t notice the toolchain support. Will amend.